cpp/data/system_updates/1.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+install --owner=root --group=root --mode=644 /usr/local/ub_tools/cpp/data/rsyslog.conf /etc/rsyslog.d/ub_tools.conf
```

Comment:
this should also be performed by installer.cc & system_version should be detected by installer